finance-tracker/app/views/layouts/_navigation.html.erb

```diff
@@ -12,6 +12,9 @@
           </li>
         </ul>
         <ul class="navbar-nav ml-auto">
+          <li class="nav-item user-name">
+            <%= fa_icon 'user' %> <%= current_user.full_name %>
+          </li>
```

Comment:
What does this have to do with adding first and last name column into user table?